security/README.md

```diff
@@ -21,35 +21,22 @@ assets of Sparkbox and its clients.  When in doubt about any matter related to
 security, employees should communicate quickly and clearly in the #security
 Slack channel.
 
-## Instructions
+## Table of Contents
 
-The following guides provide instructions for checking the status of and 
-implementing important aspects of the Sparkbox Security Policy. 
+### Security Standards
 
-Screenshots can be captured using [macOS][macOS screenshots], [iOS][iOS
-screenshots], or Android. Reach out to the #security Slack channel with 
-questions or challenges.
+- **[Security Principles](security_principles)**
+- **Frontend:**
+  - **[JavaScript](frontend/javascript)**
+  - **[Exception Management](frontend/exception_management)**
+- **Backend:**
+  - **[Access Control](backend/access_control)**
+  - **[File Management](backend/file_management)**
+  - **[Deployment](backend/deployment)**
```

Comment:
Can _Deployment_ be a top level, sibling of _Frontend_ and _Backend_?

Reply:
Yes, good call!